projects/kit/components/rating/rating.template.html

```diff
@@ -17,13 +28,27 @@
         >
             <tui-svg
                 *tuiRepeatTimes="let index of max"
-                class="t-icon-filled"
+                class="t-icon-rating t-icon-filled"
+                [class.t-icon-rating-first]="max - 1 === index"
+                [class.t-icon-rating-last]="index === 0"
                 [src]="iconFilled"
-            ></tui-svg>
+            >
+                <!--
+                    @Description:
+                    [class.t-icon-rating-first]="last"
+                    [class.t-icon-rating-last]="first"
+
+                    The last and the first changed in places,
+                    since we specifically reverse the elements by CSS
+                -->
```

Comment:
nit: I don't think we need those comments in template

Reply:
fixed